crates/engine/tree/Cargo.toml

```diff
@@ -31,7 +31,7 @@ reth-stages-api.workspace = true
 reth-tasks.workspace = true
 reth-trie-db.workspace = true
 reth-trie-parallel.workspace = true
-reth-trie-sparse.workspace = true
+reth-trie-sparse = { workspace = true, features = ["std", "metrics"] }
```

Comment:
@shekhirin I enabled the metrics feature here so that we still record them.

I also forward activated them in trie-parallel but I noticed we never activate the metrics feature for trie-parallel?